pkg/gits/git_cli.go

```diff
@@ -784,8 +784,23 @@ func (g *GitCLI) GetRemoteUrl(config *gitcfg.Config, name string) string {
 
 // RemoteBranches returns all remote branches with the given prefix
 func (g *GitCLI) RemoteBranchNames(dir string, prefix string) ([]string, error) {
+	return g.remoteBranchNames(dir, prefix, false)
+}
+
+// RemoteMergedBranchesNames returns all remote branches that were merged with the given prefix
+func (g *GitCLI) RemoteMergedBranchNames(dir string, prefix string) ([]string, error) {
```

Comment:
_prefix_ seems to be a bit generic as a function parameter name in this context. Maybe `remoteBranchFilter`, `remoteRefPrefix` or something similar which comes out of the git domain